railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -56,9 +56,14 @@ Rails.application.configure do
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   # config.force_ssl = true
 
-  # Include generic and useful information about system operation, but avoid logging too much
-  # information to avoid inadvertent exposure of personally identifiable information (PII).
-  config.log_level = :info
+  # Log to STDOUT by default
+  config.logger = ActiveSupport::Logger.new(STDOUT)
+    .tap  { |logger| logger.formatter = ::Logger::Formatter.new }
+    .then { |logger| ActiveSupport::TaggedLogging.new(logger) }
+
+  # Use the lowest log level to ensure availability of diagnostic information
+  # when problems arise.
+  config.log_level = :debug
```

Comment:
This is a big change, so it should be in the CHANGELOG. Also I'm not sure if it is by design, so I'm calling this out. We used to have `:debug` as default, [them we make `:info` the default](https://github.com/rails/rails/commit/229fd2a02fc694b4b7756445b6647777aa94e25d). Now we are making `:debug` default again.